test/AWSUtils.Tests/AWSUtils.Tests.csproj

```diff
@@ -129,6 +129,7 @@
     <Compile Include="CollectionFixtures.cs" />
     <Compile Include="LivenessTests.cs" />
     <Compile Include="MembershipTests\DynamoDBMembershipTableTest.cs" />
+    <Compile Include="MembershipTests\SiloInstanceRecordTests.cs" />
```

Comment:
Please add this test into the `vNext/test/AWSUtils.Tests`... It is a link to this file so the test can run in .net core. I've added it here and it pass without problem.